### PR TITLE
Desire

### PR DESCRIPTION
**Pull request by NadiaHG**
Setup desire configuration files

### CODE REVIEW

**Comment by abubeck** [comment]
Can you please update the desire.launch file to implement the new xml + launch file structure.

File: cob_hardware_config/cob3-2/config/light.yaml [hunk 1]
```diff
@@ -1,2 +1,6 @@
 devicestring: /dev/ttyUSB0
 baudrate: 230400
+<<<<<<< HEAD
```
Comment (floweisshardt) [inline cob_hardware_config/cob3-2/config/light.yaml:3]: please resolve merge conflict

File: cob_bringup/robots/cob3-6.xml [hunk 2]
```diff
@@ -5,8 +5,11 @@
 	<arg name="pc2" default="localhost"/>
 
 	<group>
+
+		<machine name="$(arg pc1)" address="$(arg pc1)" default="true"/>
```
Comment (floweisshardt) [inline cob_bringup/robots/cob3-6.xml:9]: should be <machine name="pc1" address="$(arg pc1)" default="true"/>

File: cob_bringup/robots/cob3-6.xml [hunk 3]
```diff
@@ -40,8 +43,10 @@
 	</group>
 
 	<group>
+		<machine name="$(arg pc2)" address="$(arg pc2)" default="true"/>
 		<machine name="pc2" address="$(arg pc2)" default="true"/>
```
Comment (floweisshardt) [inline cob_bringup/robots/cob3-6.xml:47]: should be <machine name="pc2" address="$(arg pc2)" default="true"/>

File: cob_bringup/robots/cob3-6.xml [hunk 4]
```diff
@@ -50,10 +55,15 @@
 		<!-- start camera sensors -->
 		<include file="$(find cob_bringup)/components/prosilica_left.launch" />
 		<include file="$(find cob_bringup)/components/prosilica_right.launch" />
+		<include file="$(find cob_sound)/ros/launch/sound.launch" />
+	</group>
+
+	<machine name="$(arg pc1)" address="$(arg pc1)" default="true"/>
```
Comment (floweisshardt) [inline cob_bringup/robots/cob3-6.xml:61]: should be <machine name="pc1" address="$(arg pc1)" default="true"/>

**Comment by floweisshardt** [comment]
please fix issues and send a pull request again

**Pull request closed without merging**